please

```diff
@@ -176,8 +181,15 @@ doctor() {
         FAIL="."
     fi
 
+    if _hasKvmSupport
+    then
+        echo -e "- kvm support: $OK"
+    else
+        echo -e "- kvm support: '/dev/kvm' not found. You probably won't be able to start VMs"
```

Comment:
The error message is not correct since this can also occur if permissions are not corrects.

Reply:
Ooops. right :-)

Reply:
Thanks for catching this. I updated the message